static/googlebd66caa1867bee6e.html

```diff
@@ -0,0 +1 @@
+google-site-verification: googlebd66caa1867bee6e.html
```

Comment:
@vipulgupta2048 according to https://docusaurus.io/docs/static-assets, files placed into `static/` should be copied into the build folder. But the site resulting from this PR only contains the following assets:

```
File name
404.html
index.html
search-index-docs-default-current.json
sitemap.xml

BUILD
index.html

assets

css

js

img
favicon.ico
logo.png
```

Any ideas?

Reply:
Maybe: https://github.com/product-os/docusaurus-builder/pull/56 ?

Reply:
closing for now, will PR custom Docusaurus config feature later.